components/event-handler-notification/org.wso2.carbon.identity.event.handler.notification/src/main/java/org/wso2/carbon/identity/event/handler/notification/util/NotificationUtil.java

```diff
@@ -149,6 +150,27 @@ public static Map<String, String> getPlaceholderValues(EmailTemplate emailTempla
 
         Map<String, String> configFilePlaceholders = getConfigFilePlaceholders();
 
+        JsonNode brandingPreferences = null;
+        Map<String, String> brandingFallbacks = getBrandingFallbacksFromConfigFile();
+
+        if (Boolean.parseBoolean(
+                IdentityUtil.getProperty(NotificationConstants.EmailNotification.ENABLE_ORGANIZATION_LEVEL_EMAIL_BRANDING))) {
+            try {
+                BrandingPreferenceManager brandingPreferenceManager = new BrandingPreferenceManagerImpl();
+                BrandingPreference responseDTO = brandingPreferenceManager.getBrandingPreference("ORG", placeHolderData.get("tenant-domain"), "en-US");
```

Comment:
Lets' break this line since it goes beyond 120 characters.

Reply:
fixed with 3d35c2fe0ace0082531c28534363927af174f71f